pkg/v3/config/config.go

```diff
@@ -63,6 +63,20 @@ type OffchainConfig struct {
 
 	// MaxUpkeepBatchSize is the max upkeep batch size of the OCR2 report.
 	MaxUpkeepBatchSize int `json:"maxUpkeepBatchSize"`
+
+	// LogProviderConfig holds configuration for the log provider
+	LogProviderConfig LogProviderConfig `json:"logProviderConfig"`
+}
+
+type LogProviderConfig struct {
+	// NumOfLogUpkeeps is the number of log upkeeps supported by the registry
+	NumOfLogUpkeeps uint32 `json:"numOfLogUpkeeps"`
+
+	// FastExecLogsHigh is the maximum number of logs that we are committed to processing for each upkeep
+	FastExecLogsHigh uint32 `json:"fastExecLogsHigh"`
```

Comment:
where do we define the default values for these?

Reply:
Can we define these concretely. Are these per block / per second. Do we need float values

Reply:
Depending on this we might need to add a block time config.

Reply:
The idea was that these are per block, so yes we need to use floats.
Block time is not needed as we can calculate it.